@FISHDATA/personalAnalysis.m

```diff
@@ -1,7 +1,8 @@
 % obj personal analysis to see whether obj
-% learned the task and to measure performance 
+% learned the task and to measure performance
 % from several perspectives
 % idxPostTr: indices of post trials for test
+% TODO: rewrite this function to fit the current trial definition
 function TrRes = personalAnalysis(obj,idxPostTr)
 
 TrRes = zeros(24,5);
```

Comment:
Better to write/Use a function to replace all lines for plotting though there are just several lines.